tests/test_auto_dispute_multiple_ids.py

```diff
@@ -0,0 +1,215 @@
+    assert btc_timestamp[2] > 0
+    assert status.ok, status.error
+
+    with patch("getpass.getpass", return_value=""):
+        with patch("tellor_disputables.alerts.send_text_msg", side_effect=print("alert sent")):
+            with patch("tellor_disputables.cli.TelliotConfig", new=lambda: config):
+                with patch("telliot_feeds.feeds.evm_call_feed.source.cfg", config):
+                    try:
+                        async with async_timeout.timeout(9):
+                            await start(False, 8, "disputer-test-acct", True, 0.1)
+                    except asyncio.TimeoutError:
+                        pass
+    indispute, _ = await oracle.read("isInDispute", eth_query_id, eth_timestamp[2])
+    assert indispute
+    # btc value should not be disputed since not in config
+    indispute, _ = await oracle.read("isInDispute", btc_query_id, btc_timestamp[2])
+    assert not indispute
+    indispute, _ = await oracle.read("isInDispute", evm_query_id, evm_timestamp[2])
+    assert indispute
+
+
+@pytest.mark.asyncio
+async def test_custom_btc_config(environment_setup):
+    config, oracle, w3 = await environment_setup
+    chain_timestamp = w3.eth.get_block("latest")["timestamp"]
+    eth_timestamp, status = await oracle.read("getDataBefore", eth_query_id, chain_timestamp)
+    assert status.ok, status.error
+    evm_timestamp, status = await oracle.read("getDataBefore", evm_query_id, chain_timestamp + 5000)
+    assert evm_timestamp[2] > 0
+    assert status.ok, status.error
+    btc_timestamp, status = await oracle.read("getDataBefore", btc_query_id, chain_timestamp + 10000)
+    assert btc_timestamp[2] > 0
+    assert status.ok, status.error
+
+    btc_config = {"feeds": [{"query_id": btc_query_id, "threshold": {"type": "Percentage", "amount": 0.75}}]}
+    with patch("getpass.getpass", return_value=""):
+        with patch("tellor_disputables.alerts.send_text_msg", side_effect=print("alert sent")):
+            with patch("builtins.open", side_effect=custom_open_side_effect):
+                with patch("yaml.safe_load", return_value=btc_config):
+                    with patch("tellor_disputables.cli.TelliotConfig", new=lambda: config):
+                        try:
+                            async with async_timeout.timeout(9):
+                                await start(False, 8, "disputer-test-acct", True, 0.1)
+                        except asyncio.TimeoutError:
+                            pass
+    indispute, _ = await oracle.read("isInDispute", btc_query_id, btc_timestamp[2])
+    assert indispute
+    indispute, _ = await oracle.read("isInDispute", eth_query_id, eth_timestamp[2])
+    assert not indispute
+    indispute, _ = await oracle.read("isInDispute", evm_query_id, evm_timestamp[2])
+    assert not indispute
+
+
+@pytest.mark.asyncio
+async def test_custom_eth_btc_config(environment_setup):
+    """Test that eth and btc in dispute config"""
+    config, oracle, w3 = await environment_setup
```

Comment:
Lines 183-192 are repeated a few times in these different test functions. Consider reducing the code duplication by putting them in a function or fixture